ur5_teleop_config:
  teleop-active:
    True
  workspace:
    center:
      [0.05, 0.35, 1.20]
    scale:
      [0.5, 0.3, 0.25]